src/main/site/assets/less/layout.less

```diff
@@ -30,10 +30,6 @@ body {
   padding: 0;
 }
 
-.holder {
```

Comment:
.holder comes from src/main/resources/main.tpl .
I think it's ok to remain that class in the template for future usage,  but obviously the empty block should be removed from css.